quickwit/quickwit-proto/src/indexing/mod.rs

```diff
@@ -207,6 +207,19 @@ impl TryFrom<&str> for IndexingTask {
     }
 }
 
+#[derive(Debug, Clone, Copy, Default, Eq, PartialEq, Serialize, Deserialize, utoipa::ToSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum IndexingStatus {
+    #[default]
+    Active,
+    // We have emitted all documents of the pipeline until EOF.
```

Comment:
I think that's dangerous. Why don't we set that status on `suggest_truncate` rather than on receiving an EOF record?

Reply:
This was the previous semantic for `Complete`. We can change it if you think it is safe.